mentorApp/lib/FTU/Filters/DropDownPage.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:mentorApp/FTU/Filters/DropDownButton.dart';
+
+class DropDownPage extends StatelessWidget {
+  //static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+        backgroundColor: Colors.transparent,
+        elevation: 0.0,),
+        body: Center(
+          child: MyStatefulWidget(),
+        ),
+        floatingActionButton: FloatingActionButton(
+          child: Text("Next"),
+          onPressed: (){
```

Comment:
I'll have to test this out tomorrow, but I believe you could just call `Completion()` here to load the screen.